### DISCUSSION TITLE
can i play one song on every hour?

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#6197] [General]
can i play one song on every hour? 
it would be nice if it can.

**Comment by Mozitux** [comment]
Do you want to play one song at the end or beginning of the hour, like a jingle?

**Comment by ghost** [comment]
At the begin

**Comment by ghost** [comment]
its like a update

**Comment by ghost** [comment]
hello?

**Comment by paulcjkp** [comment]
You can create a playlist, and switch it from General Rotation, to Play Once Every x Minutes, and set the time to 60 minutes.  Then, check the box for "Play only one track".

To set it at the start of every hour, I believe you can go to the schedule page, and set the time for 12:00 am to 11:59pm, set for all days (or your desired timeframe).
